### DISCUSSION TITLE
I forgot my Gmail password

### DISCUSSION THREAD

**Discussion by Loason777** [community/community#144770] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I forgot my Gmail password,heelppp

**Comment by imDarshanGK** [comment, marked as answer]
another email id add to the recovery email id @Loason777

**Reply by Loason777** [reply]
I dont know how?

**Reply by dzinnnn** [reply]
https://support.google.com/accounts/answer/7682439?hl=en

**Reply by Loason777** [reply]
Thanks but not working..
I have Fsb profile, 14years,and my mail, username i have that pasword, a see all notification. But i cant Log In, because mail for verification code is Gmail, and i cant remember pasword, i los phone number.. Thats posibile for fix? Thanks people

**Reply by dzinnnn** [reply]
Fsb is Facebook?
1. Open a browser and go to Facebook Account Recovery. Enter your Facebook username, phone number, or alternate email address linked to the account. Click "Search" to locate your account. 2. When prompted to receive a recovery code, click "No longer have access to these?" to explore alternative recovery options. 3. Facebook will ask for verification details like an old password, your full name, names of friends, or groups you’ve joined. If none work, Facebook might request a government ID. Prepare an ID like a passport or driver’s license and follow the instructions to upload it. 4. Wait for Facebook to review your case, which can take a few hours to a few days. If approved, Facebook will send a recovery link to a new email or phone number you provide. 5. After regaining access, go to "Settings & Privacy > Settings > Contact Information" to add a new email or phone number. Verify it and remove the old Gmail address from your account.

**Reply by Loason777** [reply]
Thanks

**Comment by Akash1134** [comment]
Hi @Loason777 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
